### ISSUE TITLE
[Bugs]Few new bugs in 1.67b

### ISSUE THREAD

**Issue by moepi2k** [suriyun-production/mmorpg-kit-docs#1093]
1) When player drop an item and another one want to pick it up he gets the message "not allowed to loot" seems like its locked, but its not.

2) Healing Skill no longer works. you apply the buff but it not recover anything. (right now i only noticed taht with healing, i dont know if others also broken)

**Comment by Sour-J** [comment]
C is out try that and see if its fixed ill also test on my end but B is bugged a bit

**Comment by moepi2k** [comment]
i dont think its fixxed in C also

**Comment by insthync** [comment]
To fix healing skill, 

in `BaseCharacterEntity_BuffFunctions` line: `103` changes to
```
            if (newBuff.GetDuration() <= 0f)
            {
                CharacterRecoveryData recoveryData = new CharacterRecoveryData(this, buffApplier);
                recoveryData.Setup(newBuff);
                recoveryData.Apply(1f);
            }
```

**Comment by Sour-J** [comment]
i also have the same issue and the code is throwing an error for me :(

**Comment by insthync** [comment]
Which errror ?

**Comment by moepi2k** [comment]
here everythings fine after adding fix

**Comment by Sour-J** [comment]
When I replace what you said above using 1 67c it gives me error saying
"expected }"
But the last bracket is correct lol 😆 I'll have to try again

**Comment by insthync** [comment]
Fixed in 1.68
